SharedCode/src/commonMain/kotlin/StationHelper.kt

```diff
@@ -0,0 +1,30 @@
+package com.jetbrains.handson.mpp.mobile
+
+import io.ktor.client.*
+import io.ktor.client.features.json.*
+import io.ktor.client.features.json.serializer.*
+import io.ktor.client.request.*
+import kotlinx.serialization.json.Json
+
+class StationHelper {
+    companion object {
+
+        private val baseURL = "https://mobile-api-softwire2.lner.co.uk/"
```

Comment:
I think you are declaring the same thing in both places so I think it might be worth saving it in a common place. furthermore this might be a sign that some function that uses it is not in the correct file. m(as in everything to do with the url should probs be together)